src/main/java/com/zendesk/maxwell/schema/Database.java

```diff
@@ -48,6 +49,24 @@ public Table findTable(String name) {
 		return null;
 	}
 
+	public Table findTableOrThrow(String table) throws SchemaSyncError {
+		Table t = findTable(table);
+		if ( t == null )
+			throw new SchemaSyncError("Couldn't find table '" + table + "'" + " in database '" + this.name);
```

Comment:
if you look at the shape of the calling code, it's pretty common that we have a database in hand, and then need to check if the table exists, or what have you.  We could maybe just rename `SchemaSyncError` to `InvalidSchemaError`?